server/utils/logger.ts

```diff
@@ -0,0 +1,10 @@
+import { createConsola } from 'consola';
+
+const logger = createConsola({
+  defaults: {
+    tag: 'Vidur',
+  },
```

Comment:
```suggestion
  defaults: {
    tag: 'Vidur',
  },
```

this will sure vidur logs

```bash
[Vidur] ℹ Seeding Database
[Vidur] ◐ Connecting to PostgreSQL
ℹ Vite server warmed up in 3421ms                                                                                                                                 8:12:02 am
[Vidur] ✔ Connected to PostgreSQL
[Vidur] ℹ Current Seed Version 3 Total Seed Versions 3
[Vidur] ℹ Mounting files on ./.data
[Vidur] ℹ Mounting local driver.
[Vidur] ℹ Seeding Cache
[Vidur] ℹ Reusing existing database connection.
[Vidur] ◐ Checking if access key present
[Vidur] ℹ Access key not found, it means vidur instance is setup
[Vidur] ℹ Reusing existing database connection.
[Sun Mar 30 2025 08:12:05 GMT+0530 (India Standard Time)]: Marked 0 postings as expired.
[Vidur] ℹ [PUBLIC] postings page found 0 postings.
[Vidur] ℹ grabbing asset ce66f077-c7da-45a6-b49c-97ab25b40e57
```